bug_replication/bard1.jl

```diff
@@ -0,0 +1,49 @@
+# # bard1.mod	QQR2-MN-8-5
```

Comment:
What do these `bug_replication` files do? Should these be part of `test`?

Reply:
I noticed this after my most recent commit and push. I deleted the file and pushed that version to the wrong branch. Fixed that issue.